test/0.8.9/unstructured-storage.test.sol

```diff
@@ -0,0 +1,134 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.4.24 <0.9.0;
+
+import "forge-std/Test.sol";
+import { UnstructuredStorage } from "contracts/0.8.9/lib/UnstructuredStorage.sol";
+
+contract ExposedUnstructuredStorage {
+    function _getStorageBool(bytes32 position) public returns (bool) {
+        return UnstructuredStorage.getStorageBool(position);
+    }
+
+    function _getStorageAddress(bytes32 position) public returns (address) {
+        return UnstructuredStorage.getStorageAddress(position);
+    }
+
+    function _getStorageBytes32(bytes32 position) public returns (bytes32) {
+        return UnstructuredStorage.getStorageBytes32(position);
+    }
+
+    function _getStorageUint256(bytes32 position) public returns (uint256) {
+        return UnstructuredStorage.getStorageUint256(position);
+    }
+
+    function _setStorageBool(bytes32 position, bool data) public {
+        return UnstructuredStorage.setStorageBool(position, data);
+    }
+
+    function _setStorageAddress(bytes32 position, address data) public {
+        return UnstructuredStorage.setStorageAddress(position, data);
+    }
+
+    function _setStorageBytes32(bytes32 position, bytes32 data) public {
+        return UnstructuredStorage.setStorageBytes32(position, data);
+    }
+
+    function _setStorageUint256(bytes32 position, uint256 data) public {
+        return UnstructuredStorage.setStorageUint256(position, data);
+    }
+}
+
+contract ExposedUnstructuredStorageTest is Test {
+    ExposedUnstructuredStorage public unstructedStorage;
+
+    function setUp() public {
+        unstructedStorage = new ExposedUnstructuredStorage();
+    }
+
+    function testGetStorageBool() public {
+        bytes32 position = keccak256("FOO"); 
+        assertEq(unstructedStorage._getStorageBool(position), false);
+    }
+
+    function testGetStorageAddress() public {
+        bytes32 position = keccak256("FOO");
+        assertEq(unstructedStorage._getStorageAddress(position), address(0));
+    }
+
+    function testGetStorageBytes32() public {
+        bytes32 position = keccak256("FOO");
+        bytes32 data;
+        assertEq(unstructedStorage._getStorageBytes32(position), data);
+    }
+
+    function testGetStorageUint256() public {
+        bytes32 position = keccak256("FOO");
+        uint256 data;
+        assertEq(unstructedStorage._getStorageUint256(position), data);
+    }
```

Comment:
Should these methods be called like `testGetStorage${Type}Unitialized`, wdyt?

Reply:
good idea, adopted.